.trivyignore

```diff
@@ -1,4 +1,5 @@
 # our image does not use the network features of esbuild https://esbuild.github.io/api/#serve
 CVE-2024-24790
 CVE-2023-45288
-CVE-2024-34156
\ No newline at end of file
+CVE-2024-34156
+CVE-2024-21538
```

Comment:
Is this change intended?

Reply:
Yes. I'm not sure if this is the best way to modify it, but making this change stopped the Trivy test from failing = =

Reply:
Let me try to fix this (not avoiding this bug) and I will update again